localstack/testing/pytest/fixtures.py

```diff
@@ -69,17 +69,23 @@
 LOG = logging.getLogger(__name__)
 
 
-def _client(service, region_name=None):
-    if os.environ.get("TEST_TARGET") == "AWS_CLOUD":
-        return boto3.client(service, region_name=region_name)
+def _client(service, region_name=None, *, additional_config=None):
```

Comment:
restructured this a bit to allow providing a custom config for both AWS as well as LocalStack targeting clients